src/App.jsx

```diff
@@ -14,57 +15,79 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const SelectedGoods = ({ selectedGood, onClear }) => {
+  if (!selectedGood) {
+    return (
+      <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+    );
+  }
 
+  return (
     <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
+      {selectedGood} is selected
+      <button
+        onClick={onClear}
+        data-cy="ClearButton"
+        type="button"
+        className="delete ml-3"
+      />
     </h1>
+  );
```

Comment:
DRY.
Now you use `<h1 className="title is-flex is-align-items-center">` 2 times
Better to change only content with ternary operator